### ISSUE TITLE
Add package manager support

### ISSUE THREAD

**Issue by francipvb** [dkager/tolk#13]
Hello,

I found that it is not easy to add tolk into a source-only repository, because it has no support for library package managers.

Tools to be supported:

* [ ] NuGet: This is the most important because most development will be _.NET_ or _C++_ native applications and they can be developed with references to NuGet packages to avoid inclusion of libraries with the sources.
* [ ] PyPI: The second tool in importance because many blind developers develop python programs and it will be nice to have a weel or source build enabled for python package index.
* [ ] Gradle/Maven: For java packages.

Cheers,

**Comment by dkager** [comment]
You're absolutely right. I've been considering at least PyPI but never got around to look into it. All contributions welcome! :)

**Comment by vincent-lg** [comment]
Coming from Python, it's pretty hard to compile the library as it is.  I don't lack low-level tools, but I'm not used to compiling C/C++ code, especially on Windows without a nice CLI.  Having Tolk on Pypi would be great, or even having Tolk.dll in the libs folder, since (I imagine) it doesn't need to be recompiled on every machine.
Thanks!
Vincent

**Comment by cyrmax** [comment]
Looks like it is so quiet here...
I was looking for tolk on nuget but only found this topic. Sad, very sad.

**Comment by beqabeqa473** [comment]
just install compiler linker use nmake and you'll have a luck.


On 2/18/24, Kirill Belousov ***@***.***> wrote:
> Looks like it is so quiet here...
> I was looking for tolk on nuget but only found this topic. Sad, very sad.
>
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/dkager/tolk/issues/13#issuecomment-1951335022
> You are receiving this because you are subscribed to this thread.
>
> Message ID: ***@***.***>


-- 
with best regards Beqa Gozalishvili
Tell: +995593454005
Email: ***@***.***
Web: https://gozaltech.org
Skype: beqabeqa473
Telegram: https://t.me/gozaltech
facebook: https://facebook.com/gozaltech
twitter: https://twitter.com/beqabeqa473
Instagram: https://instagram.com/beqa.gozalishvili
